vnext/Microsoft.ReactNative/Views/ConfigureBundlerDlg.cpp

```diff
@@ -0,0 +1,180 @@
+  }
+}
+
+void ConfigureBundlerDlg::CreateAndShowUI() noexcept {
+  if (auto reactHost = m_weakReactHost.GetStrongPtr()) {
+    const winrt::hstring xamlString =
+        LR"(
+  <StackPanel
+    xmlns='http://schemas.microsoft.com/winfx/2006/xaml/presentation'
+    xmlns:x='http://schemas.microsoft.com/winfx/2006/xaml'
+    Background='{ThemeResource ContentDialogBackground}'
+    HorizontalAlignment='Center'
+    MinWidth="{ThemeResource ContentDialogMinWidth}"
+    MaxWidth="{ThemeResource ContentDialogMaxWidth}"
+    MinHeight="{ThemeResource ContentDialogMinHeight}"
+    MaxHeight="{ThemeResource ContentDialogMaxHeight}"
+    Margin='-10'
+    Padding="{ThemeResource ContentDialogPadding}"
+  >
+    <StackPanel.Transitions>
+      <TransitionCollection>
+        <EntranceThemeTransition />
+      </TransitionCollection>
+    </StackPanel.Transitions>
+    <TextBlock FontSize="20" Margin="{ThemeResource ContentDialogTitleMargin}">Configure Bundler</TextBlock>
+    <TextBox x:Name="Host" Header="Bundler host:" PlaceholderText="localhost"/>
+    <TextBox x:Name="Port" Header="Bundler port:" PlaceholderText="8081" Margin="0,4,0,0"/>
+    <TextBox x:Name="EntryPoint" Header="entrypoint:" PlaceholderText="index.windows" Margin="0,4,0,0"/>
+    <StackPanel Orientation="Horizontal" HorizontalAlignment="Right" Margin="{ThemeResource ContentDialogCommandSpaceMargin}">
+      <Button x:Name="Apply" Style="{StaticResource AccentButtonStyle}" Content="Apply Changes" />
+      <Button x:Name="Reset" Content="Reset to Default" Margin="4,0,0,0"/>
+      <Button x:Name="Cancel" Content="Cancel" Margin="4,0,0,0"/>
+    </StackPanel>
+  </StackPanel>)";
+    auto configureUI = winrt::unbox_value<xaml::Controls::StackPanel>(xaml::Markup::XamlReader::Load(xamlString));
+
+    auto options = reactHost->Options();
+
+    configureUI.FindName(L"Host").as<xaml::Controls::TextBox>().Text(Microsoft::Common::Unicode::Utf8ToUtf16(
+        options.DeveloperSettings.SourceBundleHost.empty() ? facebook::react::DevServerHelper::DefaultPackagerHost
+                                                           : options.DeveloperSettings.SourceBundleHost));
+
+    configureUI.FindName(L"Port").as<xaml::Controls::TextBox>().Text(std::to_wstring(
+        options.DeveloperSettings.SourceBundlePort ? options.DeveloperSettings.SourceBundlePort
+                                                   : facebook::react::DevServerHelper::DefaultPackagerPort));
+
+    configureUI.FindName(L"EntryPoint")
+        .as<xaml::Controls::TextBox>()
+        .Text(Microsoft::Common::Unicode::Utf8ToUtf16(
+            options.DeveloperSettings.SourceBundleName.empty() ? "index.windows"
+                                                               : options.DeveloperSettings.SourceBundleName));
+
+    auto applyButton = configureUI.FindName(L"Apply").as<xaml::Controls::Button>();
+    m_applyRevoker = applyButton.Click(
```

Comment:
Nit: you probably don't need the revokers since configureUI will die when the Flyout is closed anyway, and the revoker dtor won't be called unless the flyout is closed.